com.unity.render-pipelines.high-definition/Documentation~/Upgrading-To-HDRP.md

```diff
@@ -8,62 +8,53 @@ This document explains how to convert your Project to HDRP.
+* You can't automatically upgrade custom Materials or Shaders to HDRP. You must [convert custom Materials and Shaders manually](#ManualConversion).
+* Height mapped Materials might look incorrect. This is because HDRP supports more height map displacement techniques and decompression options than the Built-in Render Pipeline. To upgrade a Material that uses a heightmap, modify the Material's **Amplitude** and **Base** properties until the result more closely matches the Built-in Render Pipeline version.
+* You can't upgrade particle shaders. HDRP doesn't support particle shaders, but it does provide Shader Graphs that are compatible with the [Built-in Particle System](https://docs.unity3d.com/Manual/Built-inParticleSystem.html). These Shader Graphs work in a similar way to the built-in particle shaders. To use these Shader Graphs, import the **Particle System Shader Samples** sample:
 
-This process also can't upgrade particle shaders. Even though HDRP doesn't support particle shaders, it does provide some Shader Graphs that are compatible with the [Built-in Particle System](https://docs.unity3d.com/Manual/Built-inParticleSystem.html). These Shader Graphs work in a similar way to the built-in particle shaders. To use these Shader Graphs, import the **Particle System Shader Samples** sample:
-
-1. Open the Package Manager window (menu: **Window > Package Manager**).
-2. Find and click the **High Definition RP** entry.
-3. In the package information for **High Definition RP**, go to the **Samples** section and click the **Import into Project** button next to **Particle System Shader Samples**.
+    1. Open the Package Manager window (menu: **Window** > **Package Manager**).
+    2. Find and click the **High Definition RP** entry.
+    3. In the package information for **High Definition RP**, go to the **Samples** section and click the **Import into Project** button next to **Particle System Shader Samples**.
 
 <a name="ManualConversion"></a>
 
 ### Converting Materials manually
 
-The HDRP Material converter automatically converts Built-in Standard and Unlit Materials to HDRP Lit and Unlit Materials respectively. The process uses an overlay function to blend the color channels together, similar to the process you would use in image editing software, like Photoshop. To help you convert custom Materials manually, this section describes the maps that the converter creates from the Built-in Materials.
+The automatic HDRP Material converter processes convert Built-in Standard and Unlit Materials to HDRP Lit and Unlit Materials respectively. The processes use an overlay function to blend the color channels together, similar to the process you would use in image editing software, like Adobe Photoshop. To help you convert custom Materials manually, this section describes the maps that the converter creates from the Built-in Materials.
```

Comment:
This sentence feels like it's got too many words: 

"The automatic HDRP Material converter processes convert..." very wibbly. 

Instead: 

"HDRP uses multiple processes to automatically convert Built-in Standard and Unlit Materials to HDRP Lit and Unlit Materials respectively."

Reply:
Also "These processes..." instead of "The processes" is less ambiguous.

Reply:
Remove the comma before "like Adobe Photoshop".

Reply:
I'd make "To help you convert..." The start of a new paragraph, this is a block at the moment.